File: zadania1/srednia/python/main.py
liczba_ocen = int(input("Podaj liczbę ocen: "))
suma = 0

for i in range(liczba_ocen):
    ocena = float(input("Podaj ocenę: "))
    suma = suma + ocena

srednia = suma / liczba_ocen
print("Średnia: ", srednia)

if srednia >= 3.0:
    print("Uczeń zdał")
else:
    print("Uczeń nie zdał")

File: zadania1/srednia/csharp/main.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine("Podaj liczbę ocen");
        int LiczbaOcen = int.Parse(Console.ReadLine());
        double suma = 0;

        for (int i = 0; i < LiczbaOcen; i++)
        {
            Console.WriteLine("Podaj ocenę: ");
            double ocena = double.Parse(Console.ReadLine());
            suma = suma + ocena;
        }

        double srednia = suma / LiczbaOcen;
        Console.WriteLine("Średnia wynosi: " + srednia);

        if (srednia >= 3.0)
        {
            Console.WriteLine("Uczeń zdał");
        }
        else
        {
            Console.WriteLine("Uczeń nie zdał");
        }

    }
}


